jest.config.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
   // ],
 
   // Indicates which provider should be used to instrument code for coverage
-  coverageProvider: 'v8',
+  coverageProvider: 'babel',
```

Comment:
There were a few cases that couldn't be covered (both `assertExhaustive`), and `v8` doesn't support ignoring lines for coverage, as far as I know.